src/module/pageSettings.ts

```diff
@@ -51,7 +51,9 @@ export function usePageSettings(settings?: Partial<PageSettings>, deps: Dependen
     const dispatch = useAppDispatch();
     useEffect(() => {
       dispatch(((dispatch: AppDispatch) =>
-        dispatch(setPageSettings({ ...settings, page: pathname, searchParams: {} }))) as unknown as Action);
+        dispatch(
+          setPageSettings({ ...settings, page: pathname, searchParams: new ReadonlyURLSearchParams() }),
+        )) as unknown as Action);
```

Comment:
On peut enlever le cast qui ne sert plus à rien
```suggestion
        dispatch(setPageSettings({ ...settings, page: pathname, searchParams: new ReadonlyURLSearchParams() })),
      );
```